### ISSUE TITLE
Question about main.py

### ISSUE THREAD

**Issue by huanghjA** [dair-iitd/CSSL#1]
Is the last fine-tuning training a supervised learning process?
What is the difference between DataGenerator and FT_DataGenerator?

**Comment by huanghjA** [comment]
```
metrics = {'out': dice_coef}
optimizer = Adam(learning_rate=lr)
model.compile(loss=[dice_loss], optimizer=optimizer, metrics=metrics)
```
In the fine-tuning step, I initialized the training process with the above which wasn't provided in the origin` main.py`.
Then, Train with trainset and valid with testset in the base of pre-trained model celiac_weights.h5 provided by the author.

However, I got the training log with high accuracy in trainset but extremely low accuracy in testset as following:
Epoch 5/500
2/2 [==============================] - 39s 19s/step - loss: 0.3395 - dice_coef: 0.6605 - val_loss: 0.8731 - val_dice_coef: 0.1269 - lr: 0.0010
Epoch 50/500
2/2 [==============================] - 38s 19s/step - loss: 0.0858 - dice_coef: 0.9142 - val_loss: 0.8112 - val_dice_coef: 0.1888 - lr: 0.0010
Epoch 85/500
2/2 [==============================] - 38s 19s/step - loss: 0.0493 - dice_coef: 0.9507 - val_loss: 0.9526 - val_dice_coef: 0.0474 - lr: 0.0010
Epoch 90/500
2/2 [==============================] - 37s 19s/step - loss: 0.0453 - dice_coef: 0.9547 - val_loss: 0.9607 - val_dice_coef: 0.0393 - lr: 0.0010

**Comment by AjeyPaiK** [comment]
The weights provided are already finetuned. You don't have to retrain it.

**Comment by huanghjA** [comment]
Hi, I'm trying to reimplement this repository. However, some details were lost in `main.py`, will you maintain this repository to make it run easily later.
